test/suite/test_txn02.py

```diff
@@ -201,7 +208,7 @@ def check_log(self, committed):
     def test_ops(self):
         self.backup_dir = os.path.join(self.home, "WT_BACKUP")
         self.session2 = self.conn.open_session()
-        # print "Creating %s with config '%s'" % (self.uri, self.create_params)
```

Comment:
I found these print statements useful, but turning them on, and off, and on again to keep commits clean annoying. I hope this is an agreeable, localized, alternative.